src/ordered_dict.jl

```diff
@@ -487,3 +487,11 @@ function Base.map!(f, iter::Base.ValueIterator{<:OrderedDict})
     end
     return iter
 end
+
+function last(h::OrderedDict)
+    h.ndel > 0 && rehash!(h)
+    key = h.keys[end]
+    index = ht_keyindex(h, key, false)
+    @inbounds val = h.vals[h.slots[index]]
```

Comment:
You're looking up the index where `key` is located, but don't we already know it is at `end`? Couldn't you just have `h.keys[end] => h.vals[end]`?

Reply:
Aah makes total sense!
Will make the changes.